recipe/build.sh

```diff
@@ -1,4 +1,38 @@
-./configure --prefix=$PREFIX --with-e-antic=$PREFIX --with-nauty=$PREFIX --with-flint=$PREFIX --with-gmp=$PREFIX
+#!/bin/bash
+
+set -e
+
+autoreconf --install
+chmod +x configure
+
+case `uname` in
+    Darwin|Linux)
+        export CFLAGS="-O3 -g -fPIC $CFLAGS"
+        ./configure --prefix=$PREFIX --with-e-antic=$PREFIX --with-nauty=$PREFIX --with-flint=$PREFIX --with-gmp=$PREFIX
+        ;;
+    MINGW*)
+        export PATH="$PREFIX/Library/bin:$BUILD_PREFIX/Library/bin:$RECIPE_DIR:$PATH"
+        export CC=cl_wrapper.sh
```

Comment:
You need this file from `m4ri-feedstock` and also the `bld.bat`

Reply:
Right. Forgot to add them to the index.